docs/user/integrations.rst

```diff
@@ -47,12 +47,21 @@ you will be able to preview changes and see accepted changes published immediate
 Automated versioning
 --------------------
 
-Maintaining your documentation in a git repository with Read the Docs' automated CI/CD pipeline
-also means that you will be able to align your project's **release cycle** with your documentation.
-Whenever you announce a new **version** for your documentation,
-you will probably tag that version in your git repository.
-Read the Docs automatically registers the change and adds a new version to your documentation.
-This is captured in the :term:`flyout menu`.
+With Read the Docs' automated CI/CD pipeline, you will be able to fully align your project's **release cycle** with your documentation.
+For instance, a new version of a software project can build and publish a new version of its documentation.
+
+Read the Docs is notified of all changes to your git branches and git tags.
+When you release a **new version** for your project,
+you are likely also adding a version tag to your git repository.
+
+Read the Docs is notified of changes to git tags and branches.
+These events can be configured to build and publish your documentation when new tags or branches are added in your repository.
+If you use a versioning schema, you can choose or define the schema and automate this process.
```

Comment:
```suggestion
Read the Docs is notified of all changes to your git branches and git tags.
When you release a new version for your project,
you are likely also adding a version tag to your git repository.
These git events can be configured to build and publish your documentation automatically with :doc:`/automation-rules`.
If you use a versioning schema, you can configure it as part of the automation process.
```

Reply:
This is a really good simplifying change :+1: